tests/js/spec/views/integrationPipeline/pipelineView.spec.jsx

```diff
@@ -12,13 +12,13 @@ jest.mock(
 );
 
 describe('PipelineView', () => {
-  it('renders awsLambdaProjectSelect', () => {
+  it('renders awsLambdaProjectSelect', async () => {
     mountWithTheme(
       <PipelineView pipelineName="awsLambdaProjectSelect" someField="someVal" />,
       {context: TestStubs.routerContext()}
     );
 
-    screen.findByText('mock_AwsLambdaProjectSelect');
+    await screen.findByText('mock_AwsLambdaProjectSelect');
```

Comment:
Should it be getByText if this didn't need to resolve before?

Reply:
@evanpurkhiser followed up here: https://github.com/getsentry/sentry/pull/29790